app/src/lib/stores/app-store.ts

```diff
@@ -814,6 +813,7 @@ export class AppStore {
     this.refreshMentionables(repository)
 
     this.addUpstreamRemoteIfNeeded(repository)
+    await this._refreshPullRequests(repository)
```

Comment:
I don't think we should `await` this. It means the rest of the function has to wait until it's done, and I don't think that's necessary. The store should emit an update once it's done if we need to react.